programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(numbers) {
+  var answer = new Set([]);
+  for (let i = 0; i < numbers.length; i++) {
+    for (let j = i + 1; j < numbers.length; j++) {
+      answer.add(numbers[i] + numbers[j]);
+    }
+  }
+  return [...answer].sort((a, b) => { return a - b; });
```

Comment:
익명함수(콜백)이 한 줄일때는 중괄호와 ```return```이 없어도 된다는 것 기억해주세요 :D
구조분해할당까지 아시고... 놀랍구만요!!! 1학년인데 어찌...ㅋㅋㅋㅋㅋ
```suggestion
  return [...answer].sort((a, b) => a - b);
```